### PR TITLE
add docker-compose

### PR DESCRIPTION
**Pull request by kirilmaz-k2v**

### CODE REVIEW

**Review by itsiktsarfati — APPROVED** [review]

File: Docker/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+1. Docker - [Docker install guide](https://docs.docker.com/engine/install/)
+2. Docker compose - [docker-compose install guide](https://docs.docker.com/compose/install/)
+
+## docker-compose.yaml
+This file defines and configures all the services needed for this use case. It specifies how these services are built, how they interact, and any volumes or networks they use.
+
+### environment
+This parameters can be added to services.fabric.environment .
+#### Git
+GIT_TOKEN  - Token for clone fron github repo. \
```
Comment (nader-k2view) [inline Docker/README.md:14]: Typo: fron --> from

File: Docker/fabric-studio-pg/docker-compose.yaml
Comment (nader-k2view) [inline Docker/fabric-studio-pg/docker-compose.yaml:1]: Consider adding Healthchecks for each service

Comment (kirilmaz-k2v) [reply Docker/fabric-studio-pg/docker-compose.yaml:1]: Added based on our helm livenessProbe
healthcheck:
      test: ["CMD", "/bin/bash", "/opt/apps/fabric/isAlive.sh"]
      interval: 60s
      timeout: 10s
      retries: 3
      start_period: 120s

**Review by nader-k2view — APPROVED** [review]

**Comment by nader-k2view** [comment]
Approved.

**Pull request merged**